### DISCUSSION TITLE
Issue moving to Saleor Cloud

### DISCUSSION THREAD

**Discussion by akhial** [saleor/saleor#11987] [Q&A] [answered]
Hi :wave: 

We're facing a niche issue while moving to Saleor Cloud.
Our app currently depends on setting `GRAPHQL_QUERY_MAX_COMPLEXITY` to `0` as we have some complex queries that are only run occasionally.
I understand there is no way to change this on Saleor Cloud to prevent overrunning the instance.

Are we blocked?

Thanks.

**Comment by timuric** [comment, marked as answer]
Hi @akhial please reach out via chat button in cloud console, we will help you out
